ledger-core/virtual/execute/execute.go

```diff
@@ -241,8 +241,8 @@ func (s *SMExecute) stepWaitObjectReady(ctx smachine.ExecutionContext) smachine.
 		case object.Unknown:
 			panic(throw.Impossible())
 		case object.Inactive:
-			// attempt to create object that is deactivated :(
-			panic(throw.Impossible())
+			s.prepareExecutionError(throw.E("attempt to construct object that was completely deactivated"))
```

Comment:
I am sure that it is impossible to call Constructor on Inactive object

Reply:
leave as it was: panic(throw.Impossible())